src/components/MobileNav.tsx

```diff
@@ -0,0 +1,110 @@
+import { FC } from 'react'
+import useSupabaseAuth from '../hooks/useSupabaseAuth'
+import { version } from "../../package.json";
+import { capturePostHogAnayltics } from "../lib/analytics";
+import { Menu } from '@headlessui/react'
+import { GiHamburgerMenu } from 'react-icons/gi';
+
+
+
+const navs = [
+  {
+    placeholder: 'Docs',
+    links: "https://docs.opensauced.pizza/"
+  },
+  {
+    placeholder: 'Blog',
+    links: "https://dev.to/opensauced"
+  },
+  {
+    placeholder: 'Github',
+    links: "https://github.com/open-sauced"
+  },
+  {
+    placeholder: 'YouTube',
+    links: "https://www.youtube.com/opensauced"
+  }
+]
+
+const MobileNav:FC = () => {
+  const { signIn, signOut, user } = useSupabaseAuth();
+
+  return (
+    <div className=' tablet:hidden'>
+      <Menu as="div" className="relative">
+        <Menu.Button>
+          <GiHamburgerMenu className='w-[35px] h-[35px] text-gray-600 ' />
+        </Menu.Button>
+
+        <Menu.Items className="absolute z-50 right-0 w-56 origin-top-right rounded-md shadow-sm border-gray-100 border-[1px] focus:outline-none px-1 py-1 bg-white text-sm font-semibold">
+          {
+            navs.map( nav => (
+              <Menu.Item key={nav.placeholder}>
+                <div className='bg-w p-[5px] text-[14px] text-gray150 rounded-md hover:bg-gray100 '>
+                  <a href={nav.links}>{nav.placeholder}</a>
+                </div>
+              </Menu.Item>
+            ))
+          }
+
+          {
+            !user && 
+            <Menu.Item>
+              <button
+              className='inline-block bg-white px-[15px] py-[5px] h rounded-md hover:bg-gray100 text-gray150   '
+                onClick={async () => {
+                  capturePostHogAnayltics("User Login", "userLoginAttempt", "true");
+                  await signIn({ provider: "github" });
+                }}
+                onKeyDown={async (e) => {
```

Comment:
what means `e` can you give a type ?

Reply:
@NsdHSO Yep! A type can be given there!

Reply:
I don't think this really needs a type. e is standard practice and stands for the JS Event.